kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -364,26 +585,31 @@ public static boolean exists(URI uri) {
   }
 
   /**
-   * Check if a {@link Dataset} identified by the given URI string exists.
+   * Check whether a {@link Dataset} identified by the given URI string exists.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:".
+   * URIs must begin with {@code dataset:}. The remainder of
+   * the URI is implementation specific, depending on the dataset scheme.
    *
-   * @param uri a {@code Dataset} URI string.
+   * @param uri a {@code Dataset} URI string
    * @return {@code true} if the dataset exists, {@code false} otherwise
+   * @return a {@link Collection} of Dataset names ({@link String}s)
```

Comment:
Minor: avoid breaking words with javadoc tags. Instead you can use `{@link String strings}`.

http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#link